src/services/api/NetlifyService.ts

```diff
@@ -0,0 +1,113 @@
+import axios from "axios"
+
+import { config } from "@config/config"
+
+import { isAxiosError } from "@utils/axios-utils"
+
+import logger from "@root/logger/logger"
+
+const GITHUB_ORG_NAME = config.get("github.orgName")
+const NETLIFY_ACCESS_TOKEN = config.get("netlify.accessToken")
+
+type NetlifySiteDetails = {
+  id: string
+  password: string
+  build_settings: {
+    repo_path: string
+    repo_branch: string
```

Comment:
nit: can we do a manual rename or use `lodash`'s `camelCase` function to retain the naming convention? this type seems like it's gonna be used outside of this file so it might load to the snake_case being spread.

Reply:
As discussed offline, this is contained within the file - the only export is for a `() => Promise<void>` method so it will not leak!